frontend/apps/filemanager/filemanager.lua

```diff
@@ -362,7 +362,8 @@ function FileManager:init()
     end
 
     if Device:hasKeys() then
-        self.key_events.Close = { {"Home"}, doc = "Close file manager" }
+        self.key_events.Home = { {"Home"}, doc = "go home" }
+        self.file_chooser.key_events.Close = nil
```

Comment:
I'm so used to Backspace to quit things on the emulator, which work(ed) on quite everything.
No real way to keep that while staying ok with what you want?

Reply:
Ha, I had the feeling this would not be ok.
At first, I tried to add a command parameter to reader.lua, something like "reader.lua --no-key-escape", but there were no example, so I had no idea were the variable sould be stored.
I will try to think of something.

Reply:
I'm sure @poire-z can also get used to Alt+F4 instead. :-P I think backspace or escape makes sense to quit dialogs, but I'm not so sure about the program itself.

But I am somewhat confused by some of these changes, like what is `Prec`? Could you describe what's happening right now and how these keybindings are better?

Reply:
Yes sorry, "Prec" is supposed to be "Back", it should go to upper menu or close the touchmenu.
I tried to look to make the keybindings Escape to exit but the escape key doesn't seem to be defined in `event_map_sdl2.lua `

Reply:
You can add it, of course. ;-)

```lua
[41] = "Back", -- Escape
```

(Or something like that.)